lib/pages/chat/chat_view_style.dart

```diff
@@ -14,4 +14,24 @@ class ChatViewStyle {
       EdgeInsetsDirectional.only(
         start: responsive.isMobile(context) ? 0 : 16,
       );
+
+  static BoxDecoration searchBottomViewDecoration(BuildContext context) =>
+      BoxDecoration(
+        color: Theme.of(context).colorScheme.surface,
+      );
+
+  static EdgeInsetsDirectional get noSearchResultPadding =>
+      const EdgeInsetsDirectional.all(8);
```

Comment:
you said const was better for the performance, is it as efficient as this ? 

```suggestion
  static const EdgeInsetsDirectional noSearchResultPadding = EdgeInsetsDirectional.all(8);
```

Reply:
done